### ISSUE TITLE
Is it possible to create videos longer than 60 seconds?

### ISSUE THREAD

**Issue by alexdo83** [n-eari/Video-Creation-Tool#3]
Is it possible to create videos longer than 60 seconds?

**Comment by n-eari** [comment]
yup! i should note however that this current version of the script here is 2 years old, so i imagine it wont work anymore.

**Comment by alexdo83** [comment]
Will there be an update?

**Comment by n-eari** [comment]
i have an updated and much more polished version on my local machine, but i have no plans to update this version here on github im afraid
